src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -39,12 +38,16 @@ public Optional<Movie> get(Long id) {
         try (Session session = HibernateUtil.getSessionFactory().openSession()) {
             return Optional.ofNullable(session.get(Movie.class, id));
         } catch (Exception e) {
-            throw new DataProcessingException("Can't get a movie by id: " + id, e);
+            throw new DataProcessingException("Can't get a movie by id: " + id);
         }
     }
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            return session.createQuery("from Movie ", Movie.class).list();
+        } catch (Exception e) {
+            throw new RuntimeException("Error getting all movies", e);
```

Comment:
Consider throwing a `DataProcessingException` instead of a generic `RuntimeException` to maintain consistency across your DAO implementation. This will also provide a more specific context for the error.